src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -219,7 +220,7 @@ public void execute(BaseJavaLibraryPlugin p) {
 
                                 //TODO WW, removing comparing publications from the workflow for now
                                 //by commenting out below code
-                                //t.addPublicationsComparator((PublicationsComparatorTask) task);
+                                t.addPublicationsComparator((PublicationsComparatorTask) task);
```

Comment:
sure? we can iterate on the solution some more and hook it up to the workflow when you're ready :)